### ISSUE TITLE
401 http code with invalid access token response

### ISSUE THREAD

**Issue by mikewmerritt** [akirk/enable-mastodon-apps#4]
Although authentication with my WordPress instance appears to be successful, when I try to post using Elk or Pinafore (and I think Tusky, though it's harder to see the codes and responses), I'm getting a number of 401 responses with "Invalid Access Token."

```
{
    "code": "invalid_token",
    "message": "Invalid access token",
    "data": {
        "status": 401,
        "data": []
    }
}
```

Previously I was also receiving a number of CORS errors but I believe those are resolved. Notably, the 'instance' and 'authentication' endpoints are returning this response.  There are others like 'filters' and 'custom_emojis', but I know those are not yet implemented.

The token is listen on the Mastodon Apps page and ActivityPub and Friends are installed.

**Comment by akirk** [comment]
Do you maybe also have another plugin that implements OAuth2, such as IndieAuth? I know the latter returns an error message if the token doesn't match for it, not giving other plugins the chance to authenticate them.

**Comment by akirk** [comment]
I submitted a PR at https://github.com/indieweb/wordpress-indieauth/pull/245

**Comment by mikewmerritt** [comment]
> Do you maybe also have another plugin that implements OAuth2, such as IndieAuth? I know the latter returns an error message if the token doesn't match for it, not giving other plugins the chance to authenticate them.

Yes, I do. Disabling fixed the issue. I see you put in a PR in that repo.

**Comment by akirk** [comment]
The PR was accepted but no new version of IndieAuth has been released. To help others who run into this, I'll keep this issue open until they do so.
